build-image/first-boot.bash

```diff
@@ -71,7 +71,7 @@ echo "watch cat /boot/first-boot.log" > "$HOME/.bash_profile"
 # shellcheck disable=SC2154
 if [ -z "${wifi_ssid}" ]; then
   echo "$(timestamp) [openHABian] Setting up Ethernet connection... OK"
-elif grep -q "openHABian" /etc/wpa_supplicant/wpa_supplicant.conf; then
+elif grep -q "^[[:space:]]*openHABian" /etc/wpa_supplicant/wpa_supplicant.conf && ! grep -q "dtoverlay=disable-wifi" /boot/config.txt; then
```

Comment:
not here, we just search for the keyword... it will always be commented

Reply:
Github last time showed your :space: comment to apply here hence I inserted it.
But alright then, will remove.